src/nav/new-relic-one.yml

```diff
@@ -64,7 +62,7 @@ pages:
                 pages:
                       - title: New Relic guided install overview
                         path: /docs/new-relic-one/use-new-relic-one/cross-product-functions/install-configure/new-relic-guided-install-overview
-                      - title: Install New Relic
+                      - title: Install the New Relic portfolio
```

Comment:
Is "portfolio" the right word? I've never heard New Relic referred to as a portfolio. Maybe "platform"?

Reply:
Yeah, that didn't sound right. Thanks!